Sources/AvatarView.swift

```diff
@@ -38,17 +47,19 @@ open class AvatarView: UIView {
     }
     
     convenience public init() {
-        let frame = CGRect(x: 0, y: 0, width: 30, height: 30)
-        self.init(frame: frame)
-        prepareView()
+        self.init(frame: .zero)
     }
     
     private func getImageFrom(initals: String, withColor color: UIColor = UIColor.white, fontSize: CGFloat = 14) -> UIImage {
-        _ = UIGraphicsBeginImageContext(CGSize(width: 30, height: 30))
+        let width = frame.width
+        let height = frame.height
+        if width == 0 || height == 0 {return #imageLiteral(resourceName: "Steve-Jobs")}
+        
+        _ = UIGraphicsBeginImageContext(CGSize(width: width, height: height))
         let context = UIGraphicsGetCurrentContext()!
         
         //// Text Drawing
-        let textRect = CGRect(x: 5, y: 6, width: 20, height: 20)
+        let textRect = CGRect(x: 5, y: 5, width: width - 10, height: height - 10)
```

Comment:
What happens if the Avatar is `10 x 10`?

Reply:
I didn't consider 10x10 case. I think it would be better to check width > 10 and height >10 in the above code. Instead of checking 0.